tests/integration/nm/testlib.py

```diff
@@ -25,3 +26,11 @@ def mainloop():
     yield
     success = mloop.run(timeout=5)
     assert success, 'Mainloop error: ' + mloop.error
+
+
+def mainloop_run(func):
+    @functools.wraps(func)
+    def wrapper_mainloop(*args, **kwargs):
```

Comment:
nitpick/not a must: IMHO `wrap_mainloop` or `mainloop_wrapper` would be better.

Reply:
Hmm.. This is the func that wraps the original func and which will be called every time the original is referenced.
Based on that, it should probably be a verb, like the one you suggested above.
Not sure exactly, I used the pattern usef here: https://realpython.com/primer-on-python-decorators/
Other references do not help much (https://stackoverflow.com/questions/10957409/python-naming-conventions-in-decorators).